File: UI Automation C#/Exercise MsTest V2/Exercise MsTest/Exercise MsTest/Base/UnoPages.cs
using OpenQA.Selenium;
using SeleniumExtras.PageObjects;

namespace Exercise_MsTest.Base
{
    public class UnoPages
    {
        public UnoBrowser browser;
        protected IWebDriver driver;
        public UnoPages()
        {
            browser = new UnoBrowser();
            driver = browser.CreateBrowser(UnoBrowser.Browser.Chrome);
            PageFactory.InitElements(driver, this);
        }

    }
}

File: UI Automation Java/Exercise - Maintain a POM Framework/automation_POMFramework/src/main/java/Base/UnoPages.java
package Base;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.support.PageFactory;

public class UnoPages {
	public UnoBrowser browser;
    protected WebDriver driver;
	public UnoPages(){
		browser = new UnoBrowser();
        driver = browser.CreateBrowser();
        PageFactory.initElements(driver, this);
	}

}
